tests/promela/test_model.py

```diff
@@ -0,0 +1,131 @@
+from sdl2promela.promela.model import Model
+from sdl2promela.promela.generator import generate_model
+from sdl2promela.promela.modelbuilder import *
+import os
+import io
+from tests.utils.utils import assert_lines_match
+
+TEST_DIR : str = os.path.dirname(os.path.realpath(__file__))
+RESOURCE_DIR : str = os.path.join(TEST_DIR, "resources")
+
+def read_reference(path : str):
```

Comment:
Did you consider testing framework, like `pytest`?

Reply:
Yes. It even uses pytest. Please consult the makefile...